frontend/packages/dev-console/src/components/pipelines/pipeline-overview/TriggerLastRunButton.tsx

```diff
@@ -23,8 +23,9 @@ const TriggerLastRunButton: React.FC<TriggerLastRunButtonProps> = ({
   const { label, callback, accessReview } = rerunPipelineAndStay(PipelineModel, latestRun);
```

Comment:
Could you also change this to `PipelineRunModel` I missed it in my original review of this code. We are not passing a Pipeline, but a Pipeline Run.